nixos/modules/programs/goldwarden.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, config, pkgs, ... }:
+let
+  cfg = config.programs.goldwarden;
+in
+{
+  options.programs.goldwarden = {
+    enable = lib.mkEnableOption "Goldwarden";
+    package = lib.mkPackageOption pkgs "goldwarden" {};
+  };
+
+  config = lib.mkIf cfg.enable {
+    environment.systemPackages = [
+      # for cli and polkit action
+      cfg.package
+      # binary exec's into pinentry which should match the DE
+      pkgs."pinentry-${config.programs.gnupg.agent.pinentryFlavor}"
```

Comment:
Does this need to be in `environment.systemPackages`?
In my personal setup, adding a `path = [ pkgs."pinentry-${config.programs.gnupg.agent.pinentryFlavor}" ];` to the service also worked.

Reply:
It is also used in the GUI and sometimes the client also starts the daemon on it's own and when it then is missing, things break.